ui/components/MesheryPatterns.js

```diff
@@ -483,6 +503,60 @@ function MesheryPatterns({
     );
   };
 
+  const handleVerify = (e, pattern_file, pattern_id) => {
```

Comment:
@MUzairS15 , I believe this is same as the one that I have written in MeshMap. In that case, let us just keep one copy of this and reuse it. There should be a function that verifies a pattern file using the new validation endpoint and it should be reused by both Meshery UI and MeshMap